README.md

```diff
@@ -62,28 +62,70 @@ tt convert -i <input .scc file> -o <output .ttml file>
 
 ### Command line
 
-```
-tt convert [-h] -i INPUT -o OUTPUT [--itype ITYPE] [--otype OTYPE] [--config CONFIG] [--config_file CONFIG_FILE]
-```
+`tt convert [-h] -i INPUT -o OUTPUT [--itype ITYPE] [--otype OTYPE] [--config CONFIG] [--config_file CONFIG_FILE]`
 
 * `--itype`: `TTML` | `SCC` | `STL` | `SRT` (extrapolated from the filename, if omitted)
 * `--otype`: `TTML` | `SRT` | `VTT` (extrapolated from the filename, if omitted)
 * `--config` and `--config_file`: JSON dictionaries with the following members:
-
   * `"general"."progress_bar": "true" | "false"`: whether a progress bar is displayed
   * `"general"."log_level": "INFO" | "WARN" | "ERROR"`: logging level
-  * `"imsc_writer"."time_format": "frames" | "clock_time" | "clock_time_with_frames" (default: "clock_time")`: output TTML expressions in frames (`f`), as `HH:MM:SS.mmm` or as `HH:MM:SS:FF`
-  * `"imsc_writer"."fps": "<num>/<denom>"`: specifies the frame rate _num/denom_ when `time_format` is `frames` or `clock_time_with_frames`
-  * `"stl_reader"."disable_fill_line_gap" : "true" | "false" (default: "false")`: "true" means that the STL reader does not fill gaps between lines
-  * `"stl_reader"."disable_line_padding" : "true" | "false" (default: "false")`: "true" means that the STL reader does not add padding at the begining/end of lines
-  * `"stl_reader"."program_start_tc" : "TCP" | "HH:MM:SS:FF" (default: "00:00:00:00")`: specifies a starting offset, either the TCP field of the GSI block or a user-specified timecode
-  * `"stl_reader"."font_stack" : [<font-families>](https://www.w3.org/TR/ttml2/#style-value-font-families) (default: Verdana, Arial, Tiresias, sansSerif)`: overrides the font stack
-  * `"stl_reader"."max_row_count" : "MNR" | integer (default: 23)`: specifies a maximum number of rows for open subtitles, either the MNR field of the GSI block or a user-specified value
-  
+  * `"imsc_writer": JSON object`: IMSC Writer configuration (see below)
+  * `"stl_reader": JSON object`: STL Reader configuration (see below)
+
 Example:
 
 `tt convert -i <.scc file> -o <.ttml file> --itype SCC --otype TTML --config '{"general": {"progress_bar":false, "log_level":"WARN"}}'`
 
+### IMSC Writer configuration
+
+* `"time_format": "frames" | "clock_time" | "clock_time_with_frames"` \[optional\]
+
+Specifies whether the TTML time expressions are in frames (`f`), `HH:MM:SS.mmm` or `HH:MM:SS:FF`
+
```

Comment:
For readability, it may be better to use nested list items e.g. 
* `"time_format": "frames" | "clock_time" | "clock_time_with_frames"` \[optional\]
  * Specifies whether the TTML time expressions are in frames (`f`), `HH:MM:SS.mmm` or `HH:MM:SS:FF`